src/styles/components/Button.scss

```diff
@@ -25,46 +25,153 @@
         border-color: $btn-default;
         size: $btn-size;
         height: $btn-height;
+        box-shadow: none !important;
+
+        &:focus{
+            outline: none !important;
+            border: $blue-450 $border-thick double;
+        }
+    
+        &:hover{     
+            border-color: $btn-hover;
+        }
+    
+        &:active{
+            border: $blue-600;
+            background-color: $blue-100;
+        }
+    
+        &:disabled{
+            border: $gray-500 $border-default solid;
+            background-color: $gray-200 !important;
+            color: $gray-450 !important;
+            font-size: 14px;
+        }
    
-    btn-link{
+    &.btn-link{
         @include link_button;
+        border: none !important;
+        outline-style: none !important;
+
+        &:focus{
+            border: $blue-450 $border-thin solid !important;
+        }
+    
+        &:hover{     
+            color: $blue-700;
+        }
+    
+        &:active{
+            color: $blue-800;
+            background-color: transparent !important;
+        }
+    
```

Comment:
'Focus, hover and active' block is required and repeated for all variants, please check if possible to create a mixin for it like 
.btn-success {
  @include button-variant($btn-success-color, $btn-success-bg, $btn-success-border);
}
.btn-danger {
  @include button-variant($btn-danger-color, $btn-danger-bg, $btn-danger-border);
}

Reply:
Made mixin for focus.